src/Models/ShallowWaterModels/example_oneD.jl

```diff
@@ -0,0 +1,45 @@
+# # One dimensional ShallowWater example
```

Comment:
This example doesn't belong in the source code, I think it belongs in `examples/`. I recommend naming it `examples/one_dimensional_shallow_water_equation.jl`.

Reply:
I agree.  When I have something that works I will put it there.  While I'm still breaking things I'm keeping everything local.  that's me being a bit lazy but will fix this soon.